addons/project/models/project.py

```diff
@@ -1568,6 +1586,23 @@ def copy_data(self, default=None):
         defaults = super().copy_data(default=default)
         return [{k: v for k, v in default.items() if k in self.SELF_READABLE_FIELDS} for default in defaults]
 
+    @api.model
+    def new(self, values={}, origin=None, ref=None):
```

Comment:
that's awful, we can't keep that super-obscure hack

Reply:
We agree with you but we don't see another solution. 😕

Reply:
Hello @odony , do you have any alternative idea in mind?
We all agree this is awful and hacky but we unfortunately didn't find any better solution (we looked in the stack trace and that was the only valid place we found).

Thanks for your help 🙂

Reply:
> Hello @odony , do you have any alternative idea in mind?

Have you discussed the options with @rco-odoo?
There's too much code required to make this work, and it's too brittle and low-level, this is a recipe for more problems soon, we're just delaying them slightly.